camel/embeddings/open_source_embeddings.py

```diff
@@ -0,0 +1,55 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from typing import Any, List
+
+from sentence_transformers import SentenceTransformer  # type: ignore
```

Comment:
Can we remove `# type: ignore`?

Reply:
fixed, I put "sentence-transformers.*" rather than "sentence_transformers.*" in [tool.mypy.overrides], which made it  not working hhhhh